### PR TITLE
Roadmap-2024H2

### PR DESCRIPTION
**Pull request by Madelena**
Added blog post for the Home Assistant roadmap 2024 Year-end update!

## Summary by CodeRabbit

- **New Features**
	- Introduced a roadmap update detailing progress and future priorities for Home Assistant in the second half of 2024.
	- Enhanced automation organization with user-defined categories and labels.
	- Improved dashboard layout for easier customization and relevance.
	- Upgraded voice assistant features, including area awareness and media control.
	- Commitment to supporting open protocols like Z-Wave, Zigbee, and Matter, with ongoing advancements.

- **Documentation**
	- Added a new post titled "Roadmap 2024 Year-end Update: Full steam ahead!" outlining key initiatives and community involvement.

### CODE REVIEW

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+comments: true
+categories: Announcements Roadmap
+og_image: /images/blog/2024-11-roadmap-2024h2/banner.jpg
+---
+
+<p><img src='/images/blog/2024-11-roadmap-2024h2/ha-roadmap-2024h2.png' class='no-shadow' alt='Roadmap 2024 H2 diagram, showing the priorities as a table'/></p>
+
+**TL;DR** We are making great progress on our roadmap, and moving forward onto the next priorities on the roadmap. 🚂🚃🚃🚃🚃🚃🚃🚃🚃
+
+As [we had previously mentioned](/blog/2024/06/12/roadmap-introduction/), the goal of the roadmap is to provide a North Star for all of our product initiatives to follow in a coherent and consistent direction. At the [State of the Open Home](/blog/2024/04/24/state-of-the-open-home-2024/) in April this year, we introduced to our community [our first roadmap](/blog/2024/06/12/roadmap-2024h1/). We intended to keep our community informed every half year, and this is our first update to our public roadmap.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
As [we had previously mentioned](/blog/2024/06/12/roadmap-introduction/), the goal of the roadmap is to provide a North Star for all of our product initiatives to follow in a coherent and consistent direction. At the [State of the Open Home](/blog/2024/04/24/state-of-the-open-home-2024/) in April this year, we introduced to our community [our first roadmap](/blog/2024/06/12/roadmap-2024h1/). We intend to keep our community informed every half year, and this is our first update to our public roadmap.
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+#### Newly in progress: Make default dashboards more useful and relevant
+
+Now that we’ve made customization of dashboards easier than ever, what’s next? We understand that it can be intimidating for users when they are faced with a blank slate. Some may be full of ideas on how they would like to fill their dashboards up, but some would prefer if Home Assistant came with something useful out-of-the-box. Therefore, we are now looking into making dashboards easier, even if you don’t know how to create one.
+
+![Dashboard case studies](/images/blog/2024-11-roadmap-2024h2/dashboard-casestudies.png)
+
+Over the past year, we have looked into dozens of case studies - dashboards created by many of you, from the most popular to the most pragmatic - and began to notice patterns in how our users organize their dashboards. Home Assistant is unique compared to other smart home platforms in that our dashboards are not just about controlling devices. After all, the perfect app is no app, as in, opening an app to control your light bulb is less convenient than simply flicking a light switch, or better yet, just automating it.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown:102]:
```suggestion
Over the past year, we have looked into dozens of case studies - dashboards created by many of you, from the most popular to the most pragmatic - and began to notice patterns in how our users organize their dashboards. Home Assistant is unique compared to other smart home platforms in that our dashboards are not just about controlling devices. After all, the perfect app is no app, as in, opening an app to control your light bulb is less convenient than simply flicking a light switch or, better yet, just automating it.
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+The inability to fine-tune access to Home Assistant by guests or other members of the family has been a pain point requested by many of our users. We want to investigate ways to mitigate this while improving the privacy of the household. Our research on what privacy means to users will hopefully bring more clarity to this before we start working on it.
+
+### Voice Assistants
+
+#### In good shape: Improve Assist capabilities out of the box
+
+Assist, our private voice assistant, received a lot of love over the last 6 months. Our first focus was to improve its capabilities out of the box, without needing to tinker with it. We added a lot of features that were heavily requested by the community, features that bring us one step closer to feature parity with our non-private competitors.
+
+We brought area awareness, which allows you to target devices in the same area without mentioning the area (eg. “Close the curtains” or “Turn off the lights”). We brought floor support, which allows you to target devices across areas on the same floor; very useful for homes with an open floor design (eg. “Turn on all lights on the first floor”).
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

```suggestion
We brought area awareness, which allows you to target devices in the same area without mentioning the area (like “Close the curtains” or “Turn off the lights”). We brought floor support, which allows you to target devices across areas on the same floor; very useful for homes with an open floor design (for example “Turn on all lights on the first floor”).
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+<p class='img'><lite-youtube videoid="v3mNdTsX4J0" videotitle="Voice timers with countdown text and loading bar"></lite-youtube>Timers running on an S3 Box, with countdown text and a loading bar!</p>
+
+We implemented timers, allowing any voice assistant device to be able to “ring” when a timer ends, including supporting the creation of multiple timers. You can also pause and resume them, and add or remove time from them. You can combine commands with timers by giving a delayed command, allowing you to perform an action after a specific amount of time (eg. “Turn off the TV in 30 minutes)”.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
We implemented timers, allowing any voice assistant device to “ring” when a timer ends, including supporting the creation of multiple timers. You can also pause and resume them and add or remove time from them. You can combine commands with timers by giving a delayed command, allowing you to perform an action after a specific amount of time (like “Turn off the TV in 30 minutes)”.
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+Our latest benchmark for local language models (LLMs) can be explored [here](https://github.com/allenporter/home-assistant-datasets/tree/main/reports), showcasing how models like qwen2.5 perform in realistic settings. With these insights, we’re continuing to refine LLM integrations and to ensure that LLMs can be depended upon for common use cases at home.
+
+<img src='/images/blog/2024-06-voice-chapter-7/voice-script.png' alt="Exposing voice scripts to Assist in Home Assistant Voice settings" class='no-shadow'>
+
+You can now expand LLM capabilities by [allowing them access your scripts](/blog/2024/07/03/release-20247/#expand-an-llm-using-your-scripts). This feature lets you precisely control what the LLM does in specific situations, while still benefiting from the flexibility and natural language processing that AIs excel at.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
You can now expand LLM capabilities by [allowing them to access your scripts](/blog/2024/07/03/release-20247/#expand-an-llm-using-your-scripts). This feature lets you precisely control what the LLM does in specific situations, while still benefiting from the flexibility and natural language processing that AIs excel at.
```

or 

```suggestion
You can now expand LLM capabilities by [allowing them access to your scripts](/blog/2024/07/03/release-20247/#expand-an-llm-using-your-scripts). This feature lets you precisely control what the LLM does in specific situations, while still benefiting from the flexibility and natural language processing that AIs excel at.
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
+While we are ramping up our user research pipeline, we have started working on investigating what privacy means to our users. Of the three Open Home values, privacy, choice, and sustainability, while we have a pretty solid understanding of choice and some understanding of sustainability, the idea of privacy can benefit from more clarity. We have interviewed a handful of users on this topic recently, and we can’t wait to show you what we have found.
+
+## Home Assistant Cloud
+
+Home Assistant Cloud, powered by our commercial partner Nabu Casa brings, to the many, the hard-to-reach features of Home Assistant usually reserved for power users with expensive hardware at home, with no compromise on privacy.
+
+In recent months, the Nabu Casa team has been hard at work in bringing peace of mind to using Home Assistant. For the cloud backup feature, the Home Assistant team and the Nabu Casa team are now working closely together on a comprehensive revamp of backup integrations, to create an easy one-stop solution to a good backup strategy. We are also working to ensure users a smooth onboarding for backing up to Home Assistant Cloud, as well as other third-party backup locations.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
In recent months, the Nabu Casa team has been hard at work in bringing peace of mind to using Home Assistant. For the cloud backup feature, the Home Assistant team and the Nabu Casa team are now working closely together on a comprehensive revamp of backup integrations, to create an easy one-stop solution to a good backup strategy. We are also working to ensure users have a smooth onboarding for backing up to Home Assistant Cloud, as well as other third-party backup locations.
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+
+The team had to postpone some features in favor of others like [WebRTC](/blog/2024/11/06/release-202411/#low-latency-cameras-using-webrtc). They had successfully released the infrastructure needed to support WebRTC, both STUN servers for every Home Assistant user and TURN servers for every Home Assistant Cloud subscriber, enhancing real-time communication capabilities within Home Assistant. Two-factor authentication (2FA) is one of the postponed features, but it is still in the Nabu Casa pipeline.
+
+## Open Protocols
+
+We remain committed to supporting open smart home standards, including Z-Wave Zigbee, Matter, and Thread, aligning with our Open Home mission to offer choice, privacy, and sustainability through locally controlled, cloud-independent devices.
+
+In the past six months, we have made significant steps towards certifications, ensuring our open standards implementations are up to industry standards.
+
+### Z-Wave
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown:177]: Last quarter, the focus on preparing for certification has been the biggest focus from the engineering end. Should we mention that?

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+
+As interest in Z-Wave devices in our ecosystem continues to grow, this has driven Home Assistant to advance the Z-Wave JS stack and create its own Z-Wave hardware.
+
+The work on building our own Z-Wave antenna has also been progressing smoothly. We base the design of our antenna on the [Z-Wave Reference Application Design](https://github.com/drzwave/ZRAD) to create a Z-Wave antenna with the best performance in the market.
+
+### Matter
+
+As one of the first in the industry, in June, we proudly announced that Home Assistant is now based on the [latest Matter 1.3 specification](/blog/2024/06/05/release-20246/#matter-turns-13). Since then, we continued to update our Matter implementation to improve the reliability and compatibility of Matter devices, and Matter certification should hopefully happen any day now soon!
+
+We have brought expanded device support and performance improvements to Home Assistant's Matter integration. Air purifiers, room air conditioners, and thermostats are now better supported, along with custom clusters, which allow Home Assistant to detect data changes in real time, improving network efficiency.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
We have brought expanded device support and performance improvements to Home Assistant's Matter integration. Air purifiers, room air conditioners, and thermostats are now better supported, along with custom clusters, which allow Home Assistant to detect data changes in real-time, improving network efficiency.
```

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+
+Update entities now provide firmware updates via official Matter update channels, while event entities provide more event types, allowing you to react to single press or double press events with ease. Additional features include water valve support, operation state tracking for appliances, and smoke and CO sensor integration. Matter 1.3 power and energy sensors are also supported, providing useful data for the energy dashboard.
```
Comment (frenck) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
Update entities now provide firmware updates via official Matter update channels, while event entities provide more event types, allowing you to react to single-press or double-press events with ease. Additional features include water valve support, operation state tracking for appliances, and smoke and CO sensor integration. Matter 1.3 power and energy sensors are also supported, providing useful data for the energy dashboard.
```

**Review by frenck — CHANGES_REQUESTED** [review]
One bigger comment, otherwise only nit-picks.

../Frenck

File: source/_posts/2024-11-14-roadmap-2024h2.markdown (hunk 7 above)
Comment (Madelena) [reply source/_posts/2024-11-14-roadmap-2024h2.markdown:177]: Ok! I've added a mention of that 👍

File: source/_posts/2024-11-14-roadmap-2024h2.markdown [hunk 10]
```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+title: "Roadmap 2024 Year-end Update: Full steam ahead!"
+description: "Here is an update on the progress of our 2024 roadmap and a glimpse of what's coming up next."
```
Comment (balloob) [inline source/_posts/2024-11-14-roadmap-2024h2.markdown]:
```suggestion
description: "Update on the progress of our 2024 roadmap and a glimpse of what's coming up next."
```

**Review by balloob — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Looks good to me 👍 

../Frenck

**Pull request merged**